src/pages/signing_official_console/ManageUsersDropdown.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { Button } from '@mui/material';
+import { DAA } from '../../libs/ajax/DAA';
+import { Notifications } from '../../libs/utils';
+
+export default function ManageUsersDropdown(props) {
+  const [applyAll, setApplyAll] = useState(false);
+  const [removeAll, setRemoveAll] = useState(false);
```

Comment:
These states are mutually exclusive, so you can remove one and only use the other one.